### ISSUE TITLE
Hello

### ISSUE THREAD

**Issue by rushein2013** [Treeofsavior/EnglishTranslation#1646]
Please send me Beta key for TOS , please i begging you, i want to play TOS online :(

**Comment by Sunaries** [comment]
TOS is no longer in beta. Currently the Steam version has restricted access.

**Comment by rushein2013** [comment]
so this game is not longer available to new player? am i right>? T_T

**Comment by Sunaries** [comment]
That is correct. Please read this: https://treeofsavior.com/news/?n=430&vp=2
